openeogeotrellis/backend.py

```diff
@@ -1920,20 +1956,19 @@ def as_logging_threshold_arg() -> str:
             logging_threshold = as_logging_threshold_arg()
 
             def serialize_dependencies() -> str:
-                dependencies = batch_process_dependencies or job_info.get('dependencies') or []
```

Comment:
Should only serialize SHub batch process dependencies.

We schedule those batch processes ourselves, then pass on their `source_location` (and `card4l` flag) to the batch job that is subsequently started upon their completion ("DONE"). The `load_collection`s that get invoked while evaluating the process graph in the batch job only rely on `source_location` and `card4l`, not on the arguments actually passed to `load_collection`.

In the case of unfinished job results dependencies, however, there's no need to serialize them and pass them on to the subsequent batch job, because `load_stac`s evaluated in that batch job will simply act upon the very same "running" URL that made it a dependency in the first place (but this time it will actually read the job results because they have "finished" in the meanwhile.